server/issue.go

```diff
@@ -613,12 +614,52 @@ func (s *Server) createPipelineFromIssue(ctx context.Context, issueCreate *api.I
 			}
 		}
 
+		// Find and use schema from existing tenants.
+		var schemaVersion, schema string
+		project, err := s.composeProjectByID(ctx, issueCreate.ProjectID)
+		if err != nil {
+			return nil, echo.NewHTTPError(http.StatusInternalServerError, fmt.Sprintf("Failed to fetch project ID: %v", issueCreate.ProjectID)).SetInternal(err)
+		}
+		if project.TenantMode == api.TenantModeTenant {
+			p, err := s.getTenantDatabases(ctx, issueCreate.ProjectID, m.DatabaseName)
```

Comment:
It's hard to relate why we name it p and l for database list.

(that's why I don't like golang abbr)